### PR TITLE
DAT-3381 swagger documentation, readme update

### PR DESCRIPTION
**Pull request by idradm**
https://wikia-inc.atlassian.net/browse/DAT-3381

### CODE REVIEW

File: service/template-classification-storage/src/main/java/com/wikia/services/templateclassificationstorage/domain/TemplateTypeProvider.java [hunk 1]
```diff
@@ -13,6 +14,7 @@
   private String origin;
   private Set<String> types;
 
+  @ApiModelProperty(hidden = true)
   public boolean isEmpty() {
     return this.types.size() == 0;
```
Comment (WikiaTech) [inline service/template-classification-storage/src/main/java/com/wikia/services/templateclassificationstorage/domain/TemplateTypeProvider.java]: MAJOR Use isEmpty() to check whether the collection is empty or not. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1155)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by sqreek** [comment]
LGTM

except isEmpty() from sonar

File: service/template-classification-storage/src/main/java/com/wikia/services/templateclassificationstorage/domain/TemplateTypeProvider.java (hunk 1 above)
Comment (WikiaTech) [inline service/template-classification-storage/src/main/java/com/wikia/services/templateclassificationstorage/domain/TemplateTypeProvider.java]: MAJOR Use isEmpty() to check whether the collection is empty or not. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1155)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request closed without merging**